Makefile

```diff
@@ -28,7 +28,11 @@ TEST_ENABLE_STRICT_MODE=${TESTFLAGS_ENABLE_STRICT_MODE}
 
 .PHONY: fmt
 fmt:
-	!(gofmt -l -s -d $(shell find . -name \*.go) | grep '[a-z]')
+	@echo "Verifying gofmt"
```

Comment:
Maybe reference here: 
https://github.com/etcd-io/bbolt/blob/master/scripts/fix.sh

to fix it if it fails.

Reply:
Have updated the echo to include instructions for `./scripts/fix.sh`.  Let me know if that looks ok?